### PR TITLE
[Enhance] add citation format 

### PR DESCRIPTION
**Pull request by xiliu8006**
add citation format

### CODE REVIEW

**Review by Tai-Wang — CHANGES_REQUESTED** [review]
Please also add citation format to Chinese README.

File: README.md [hunk 1]
```diff
@@ -113,6 +113,19 @@ Please refer to [getting_started.md](docs/getting_started.md) for installation.
 
 Please see [getting_started.md](docs/getting_started.md) for the basic usage of MMDetection3D. We provide guidance for quick run [with existing dataset](docs/1_exist_data_model.md) and [with customized dataset](docs/2_new_data_model.md) for beginners. There are also tutorials for [learning configuration systems](docs/tutorials/config.md), [adding new dataset](docs/tutorials/customize_dataset.md), [designing data pipeline](docs/tutorials/data_pipeline.md), [customizing models](docs/tutorials/customize_models.md), [customizing runtime settings](docs/tutorials/customize_runtime.md) and [Waymo dataset](docs/tutorials/waymo.md).
 
+## Citation
+
+If you find this project useful in your research, please consider cite:
+
+```latex
+@misc{mmdet3d2020,
+    title={{MMDetection3D}: OpenMMLab's next-generation platform for general 3D object detection},
```
Comment (ZwwWayne) [inline README.md:122]: `{MMDetection3D}: OpenMMLab's` -> `{MMDetection3D: OpenMMLab}`

File: README_zh-CN.md [hunk 2]
```diff
@@ -112,10 +112,35 @@ MMDetection3D 是一个基于 PyTorch 的目标检测开源工具箱, 下一代
...
 ## 致谢
 
 MMDetection3D 是一款由来自不同高校和企业的研发人员共同参与贡献的开源项目。我们感谢所有为项目提供算法复现和新功能支持的贡献者，以及提供宝贵反馈的用户。我们希望这个工具箱和基准测试可以为社区提供灵活的代码工具，供用户复现已有算法并开发自己的新的 3D 检测模型。
+
+## OpenMMLab 的其他项目
+
+- [MMCV](https://github.com/open-mmlab/mmcv): OpenMMLab 计算机视觉基础库
+- [MMClassification](https://github.com/open-mmlab/mmclassification): OpenMMLab 图像分类工具箱
+- [MMDetection](https://github.com/open-mmlab/mmdetection): OpenMMLab 目标检测工具箱
+- [MMDetection3D](https://github.com/open-mmlab/mmdetection3d): OpenMMLab 新一代通用 3D 目标检测平台
```
Comment (ZwwWayne) [inline README_zh-CN.md:141]: Can skip ours?

Comment (xiliu8006) [reply README_zh-CN.md:141]: Other projects did not skip themselves，so...

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**